docs/i18n-guide.md

```diff
@@ -0,0 +1,14 @@
+# Internationalization Guide (i18n guide)
+
+## Caveats
+
+1. We using [ISO 639-1][iso-639-1] two-letter code for locale directory name
+
+   But `zh` as Traditional chinese
```

Comment:
Actually not. I select `zh` not `zh-hant` (or other) cause I don't know if `zh-hans` will use fallback to `zh-hant` instead of `en`

Reply:
plz you update this file.

Reply:
I searched for `zh` in ISO 639-1, it didn't mention that `zh` means `zh-hans` so it is okay to use `zh-hant` IMO. So remove this line is enough